user-interface/src/search/SearchScreen.test.tsx

```diff
@@ -1,20 +1,24 @@
 import { MockData } from '@common/cams/test-utilities/mock-data';
-import Chapter15MockApi from '@/lib/models/chapter15-mock.api.cases';
 import { CaseBasics, CaseSummary } from '@common/cams/cases';
 import { fireEvent, render, screen, waitFor } from '@testing-library/react';
 import { BrowserRouter } from 'react-router-dom';
 import SearchScreen from '@/search/SearchScreen';
 import { CasesSearchPredicate } from '@common/api/search';
 import { buildResponseBodySuccess } from '@common/api/response';
+import Api2 from '@/lib/hooks/UseApi2';
+import testingUtilities from '@/lib/testing/testing-utilities';
+import { MockInstance } from 'vitest';
 
 describe('search screen', () => {
   let caseList: CaseSummary[];
-  const getCaseSummarySpy = vi.spyOn(Chapter15MockApi, 'get');
+  let searchCasesSpy: MockInstance;
 
   beforeEach(async () => {
     vi.stubEnv('CAMS_PA11Y', 'true');
     caseList = [MockData.getCaseSummary(), MockData.getCaseSummary()];
-    getCaseSummarySpy.mockResolvedValue(buildResponseBodySuccess<CaseBasics[]>(caseList));
+    searchCasesSpy = vi
```

Comment:
What is this doing?

Reply:
Oh.  Nevermind.  My screen made it look like there were blank lines under 19 and there aren't.  Pay no attention to my comment above :-)